oci/archive/oci_transport.go

```diff
@@ -175,7 +176,7 @@ func createOCIRef(image string) (tempDirOCIRef, error) {
 
 // creates the temporary directory and copies the tarred content to it
 func createUntarTempDir(ref ociArchiveReference) (tempDirOCIRef, error) {
-	tempDirRef, err := createOCIRef(ref.image)
+	tempDirRef, err := createOCIRefWithSystemContext(nil, ref.image)
```

Comment:
This should pass along a real context instead of `nil`.

(Apparently that already existed, with `LoadManifestDescriptor` modified previously. Where did it disappear to?)